src/sgame/sg_maprotation.cpp

```diff
@@ -1489,12 +1487,29 @@ void G_InitMapRotations()
 	{
 		Log::Warn( "%s file not found.", fileName );
 	}
+}
+
+/*
+===============
+G_InitMapRotations
+
+Load and initialise the map rotations
+===============
+*/
+void G_InitMapRotations()
+{
+	G_LoadMaprotaion( "defaultmaprotation.cfg" );
```

Comment:
What if we do: `default/maprotation.cfg`?

That's not required to make this working, but maybe we would have more default files in the future and a prefix can be useful?